Environment/Snake/snake.py

```diff
@@ -21,8 +20,8 @@ def place(self, player):
         counter = 0
         touchesSnake = True
         while counter < 1000000 and touchesSnake:
-            self.x = randint(1, 17) * 44
-            self.y = randint(1, 12) * 44
+            self.x = randint(0, (800 / 50) - 1) * self.step

```

Comment:
can this happen to be put back on a block where the snake is on already? i dont remember if snake is supposed to account fo rthat

Reply:
The "touchesSnake" logic keeps it from being placed on the snake (within the 1000000 max attempts counter)